main/ejml-dsparse/src/org/ejml/sparse/csc/CommonOps_DSCC.java

```diff
@@ -1802,6 +1804,58 @@ public static DMatrixSparseCSC apply( DMatrixSparseCSC input, DUnaryOperator fun
         return apply(input, func, input);
     }
 
+    /**
+     * This applies a given unary function on every nz row,value stored in the matrix
+     *
+     * B = f(A).   A and B can be the same instance.
+     *
+     * @param input (Input) input matrix. Not modified
+     * @param func Binary function accepting (row-index, value)
+     * @param output (Output) Matrix. Modified.
+     * @return The output matrix
+     */
+    public static DMatrixSparseCSC applyRowWise( DMatrixSparseCSC input, IDBinaryOperator func, @Nullable DMatrixSparseCSC output) {
```

Comment:
Suggestion: Maybe `applyRowIdx` or similar instead? `applyRow` for brevity? When I read row wise I think it's being feed an entire row at a time. Feel free to disagree and keep it as is.

Reply:
Also don't forget to format the file. I ended up this weekend trying to figure out how to add lint checks for this and other code maintenance issues.

Reply:
I like the `Idx` addition. Btw the whole idea is basically somewhat similar to `setNumToInd()` in Combinatorial BLAS